templates/include/md_textarea.tpl.html

```diff
@@ -0,0 +1,27 @@
+{*
+Parameters:
+  $content: The initial value for textarea
+  $issue_id: The issue id
+  $id: The id attribute to textarea
+  $name: The name attribute to textarea
+  $rows: The rows attribute to textarea
+  $cols: The cols attribute to textarea
+  $tabindex: The tabindex attribute to textarea
+  $style: The style attribute to textarea
+*}
+
+<a class="white_link" href="javascript:void(null);"
+   onClick="Eventum.hidePreview('#{$id}', '#preview')">{t}[Write]{/t}</a>
+<a class="white_link" href="javascript:void(null);"
+   onClick="Eventum.showPreview('#{$id}', '#preview', { issue_id: {$issue_id|intval|default:''} })">{t}[Preview]{/t}</a>
+<a class="white_link" href="https://commonmark.org/help/" target="_blank">{t}[Markdown Help]{/t}</a>
+<br>
+<textarea
+  id="{$id}"
+  name="{$name}"
+  rows="{$rows|default:''}"
+  cols="{$cols|default:''}"
+  tabindex="{$tabindex|default:''}"
+  style="{$style|default:''}"
+>{$content|escape:"html"}</textarea>
+<div id="preview" style="width: 97%"></div>
```

Comment:
Maybe use a unique ID like `${id}-preview` to avoid conflicts in case of multiple markdown text areas on one page. The calls to `showPreview()` and `hidePreview()` above would need to be adapted as well.

Reply:
I omitted this deliberately as:
1. it works identically to previous code.
1. this is going to go away with rewrite to prosemirror PR I will submit next

Reply:
https://github.com/eventum/eventum/pull/991